server/odc-core/src/main/java/com/oceanbase/odc/core/sql/split/SqlCommentProcessor.java

```diff
@@ -107,13 +108,8 @@ public SqlCommentProcessor(DialectType dialectType, String delimiter) {
 
     public SqlCommentProcessor() {}
 
-    public static SqlStatementIterator iterator(InputStream in, DialectType dialectType,
-            boolean preserveFormat,
-            boolean preserveSingleComments,
-            boolean preserveMultiComments,
-            Charset charset) {
-        return new SqlStatementIterator(in, dialectType, preserveFormat, preserveSingleComments, preserveMultiComments,
-                charset);
+    public static SqlIterator iterator(InputStream in, Charset charset, SqlCommentProcessor processor) {
```

Comment:
the calling way of this method is strange:`SqlCommentProcessor.iterator(..., SqlCommentProcessor)`

we may don't change the method signature.